src/data/balance/rates.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Rates for shinies and other random properties are defined in this file.
+ * CHANCE is defined as x/65536
+ * RATE is defined as 1/x
+ */
+
+/** Encounterable properties */
```

Comment:
NIT: You might be better of using the `region` feature here:

```suggestion
// #region Encounterable properties
```

---

This will also show up in the IDE minimap
<img width="141" alt="image" src="https://github.com/user-attachments/assets/c5aa0895-5a2c-4029-8916-28967f33395f">

Reply:
Did not know this was a thing in ts, cool